### PR TITLE
[Fix] - L-03 - Add better explicit docstrings

### PR DESCRIPTION
**Pull request by thedarkjester**
This PR implements issue(s) #

### Checklist

* [x] I wrote new tests for my new core changes.
* [x] I have successfully ran tests, style checker and build against my new changes locally.
* [x] I have informed the team of any breaking changes if there are any.

### CODE REVIEW

File: contracts/contracts/LineaRollup.sol [hunk 1]
```diff
@@ -74,10 +92,10 @@ contract LineaRollup is
    */
```
Comment (VGau) [inline contracts/contracts/LineaRollup.sol:92]: Please remove duplicate `NB:`  :)

File: contracts/contracts/ZkEvmV2.sol [hunk 2]
```diff
@@ -18,9 +18,13 @@ abstract contract ZkEvmV2 is Initializable, AccessControlUpgradeable, L1MessageS
   /// @dev DEPRECATED in favor of currentFinalizedState hash.
   uint256 public currentTimestamp;
 
+  /// @notice The most recent finalized L2 block number.
   uint256 public currentL2BlockNumber;
 
+  /// @notice The most recent L2 stateroot hash mapped by block number.
```
Comment (VGau) [inline contracts/contracts/ZkEvmV2.sol]: Please add a space here:  `stateroot` :)

File: contracts/contracts/lib/PauseManager.sol [hunk 3]
```diff
@@ -11,14 +11,21 @@ import { IPauseManager } from "../interfaces/IPauseManager.sol";
...
+  /// @notice This is used to pause all pausable functions.
   bytes32 public constant PAUSE_ALL_ROLE = keccak256("PAUSE_ALL_ROLE");
+
+  /// @notice This is used to unpause all unpausable functions.
   bytes32 public constant UNPAUSE_ALL_ROLE = keccak256("UNPAUSE_ALL_ROLE");
 
   // @dev DEPRECATED. USE _pauseTypeStatusesBitMap INSTEAD
   mapping(bytes32 pauseType => bool pauseStatus) public pauseTypeStatuses;
 
   uint256 private _pauseTypeStatusesBitMap;
```
Comment (VGau) [inline contracts/contracts/lib/PauseManager.sol:24]: We can also add a natpsec here :)

File: contracts/contracts/tokenBridge/TokenBridge.sol [hunk 4]
```diff
@@ -63,19 +72,24 @@ contract TokenBridge is
   // solhint-disable-next-line var-name-mixedcase
   bytes4 internal constant _PERMIT_SELECTOR = IERC20PermitUpgradeable.permit.selector;
```
Comment (VGau) [inline contracts/contracts/tokenBridge/TokenBridge.sol:74]: We can add a natspec here as well

Comment (thedarkjester) [reply contracts/contracts/tokenBridge/TokenBridge.sol:74]: done

File: contracts/contracts/lib/PauseManager.sol (hunk 3 above)
Comment (thedarkjester) [reply contracts/contracts/lib/PauseManager.sol:24]: done

File: contracts/contracts/ZkEvmV2.sol (hunk 2 above)
Comment (thedarkjester) [reply contracts/contracts/ZkEvmV2.sol]: done

File: contracts/contracts/LineaRollup.sol (hunk 1 above)
Comment (thedarkjester) [reply contracts/contracts/LineaRollup.sol:92]: this has been done on another PR

**Review by VGau — APPROVED** [review]

**Review by VGau — APPROVED** [review]

**Pull request merged**